inspirehep/base/searchext/mappings/institutions.json

```diff
@@ -102,6 +102,14 @@
                 "ICN": {
                     "type": "string",
                     "copy_to": ["affautocomplete"]
+                },
+                "address": {
```

Comment:
Can you check if it is possible to use `fields` in order to have the `address` field analyzed and a `raw` version not analyzed? And for the facets we will use `address.raw`